scenarios/kubernetes_e2e.py

```diff
@@ -473,6 +479,9 @@ def create_parser():
     parser.add_argument(
         '--kubeadm', choices=['ci', 'periodic', 'pull'])
     parser.add_argument(
+        '--kubeadm-kubernetes-version', default=None, help='Version of Kubernetes to use while '
+        'testing kubeadm (e.g. latest, latest-1.6, 1.6.4, etc)')
```

Comment:
I did consider this after I saw https://github.com/kubernetes/test-infra/pull/3082. I have been intentionally trying to keep the options to the scenario to be kubeadm-based (and keeping kubernetes-anywhere an implementation detail under the covers). It's probably worth redoing the kubetest args to be consistent with these in the future, since kubeadm is the only thing we test using kubernetes-anywhere.